doc/source/deployment/setup-deployer.rst

```diff
@@ -87,7 +75,23 @@ There are several ways to install the SUSE Containerized OpenStack software.
    b. sudo zypper install socok8s (installs to /usr/share/socok8s)
 
 
-3. Clone the `socok8s GitHub repository <https://github.com/SUSE-Cloud/socok8s>`_.
+3. (For developers only) Clone the repository.
+   
+   The following software must be manually installed on your `Deployer`:
+
+     * ansible >= 2.7.8
+     * git-core
+     * jq
+     * python3-virtualenv
+     * python3-jmespath
+     * python3-netaddr
+     * python3-openstacksdk
+     * python3-openstackclient
+     * python3-heatclient
+     * which
+
+   After the required packages are installed, clone
+   `socok8s GitHub repository <https://github.com/SUSE-Cloud/socok8s>`.
```

Comment:
All these requirements can be installed either as rpm packages, as well as using `pip` (with the appropriate env variable set for socok8s that forces creating virtual env). I believe it should be mentioned here as well

Reply:
fixed